cumulusci/cli/cci.py

```diff
@@ -216,33 +217,39 @@ def main(args=None):
         if debug:
             args.remove("--debug")
 
-        # Load CCI config
-        global RUNTIME
-        try:
-            RUNTIME = CliRuntime(load_keychain=False)
-        except Exception as e:
-            handle_exception(e, is_error_command, tempfile_path, debug)
-            sys.exit(1)
+        with set_debug_mode(debug):
```

Comment:
No strong opinion about what's best here, but there's an ExitStack you could add it to.